src/blocks/block-author-profile/styles/editor.scss

```diff
@@ -13,7 +13,7 @@
 .ab-profile-image-wrap {
     svg {
         pointer-events: none;
-        margin: 0 auto;
+        margin: auto;
```

Comment:
This looks to fix the Profile Box block issue you mentioned:

![centered-here-now](https://user-images.githubusercontent.com/4063887/83453226-ba9cc880-a41f-11ea-9416-049fbe310ba9.png)